sources/packages/backend/libs/integrations/src/cra-integration/cra-income-verification.processing.service.ts

```diff
@@ -270,12 +270,18 @@ export class CRAIncomeVerificationProcessingService {
     }
 
     try {
-      await this.craService.deleteFile(remoteFilePath);
+      // Archive file.
+      const directoryPath = path.dirname(remoteFilePath);
+      const fileBaseName = path.basename(remoteFilePath);
+      await this.craService.renameFile(
```

Comment:
Just as a suggestion, not a blocker building the archive file whole path can be a private method.